Scarab/Views/UninstallDependenciesConfirmationWindow.axaml

```diff
@@ -0,0 +1,79 @@
+<Window x:TypeArguments="viewModels:UninstallDependenciesViewModel"
+        xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:ext="clr-namespace:Scarab.Extensions"
+        xmlns:models="clr-namespace:Scarab.Models"
+        xmlns:views="clr-namespace:Scarab.Views"
+        xmlns:customControls="clr-namespace:Scarab.CustomControls"
+        xmlns:viewModels="clr-namespace:Scarab.ViewModels"
+        mc:Ignorable="d" d:DesignWidth="550"
+        x:Class="Scarab.Views.UninstallDependenciesConfirmationWindow"
+        Title="{ext:Localize XAML_UninstallDependencies_Title}"
+        SizeToContent="WidthAndHeight">
+
+  <Window.Styles>
+    <Style Selector="Button">
+      <Setter Property="MinHeight" Value="24" />
+      <Setter Property="MinWidth" Value="80" />
+    </Style>
+  </Window.Styles>
+  <Grid ColumnDefinitions="15,Auto,15,*,15"
+        RowDefinitions="15,Auto,Auto,Auto,15">
+    <Image Grid.Row="1" Grid.Column="1" Margin="0 5 0 0"
+               MaxHeight="65"
+               MaxWidth="65"
+               VerticalAlignment="Center"
+               Source="avares://MessageBox.Avalonia/Assets/stop.png"/>
+
+    <StackPanel  Grid.Row="1" Grid.Column="3" VerticalAlignment="Center">
+      <TextBlock Text="{ext:Localize XAML_UninstallDependencies}"></TextBlock>
+      <TextBlock Text="{ext:Localize XAML_ManuallyInstalledModesNote}"
+                  IsVisible="{Binding ExternalModsInstalled, Mode=OneTime}"
+                  FontStyle="Italic">
+      </TextBlock>
+    </StackPanel>
+    
+    <StackPanel Grid.Row="2" Grid.Column="3">
+     
+      <TextBlock Text="{Binding OptionsList, Mode=OneTime}"></TextBlock>
+        
+      <Expander Header="{ext:Localize XAML_ChooseDependenciesToUninstall}" Margin="0 0 0 10">
+        <StackPanel>
+          <StackPanel Orientation="Horizontal" HorizontalAlignment="Left" Spacing="10">
+            <Button Content="{ext:Localize XAML_ModsFilter_All}" Command="{Binding ToggleAll}" CommandParameter="True" MinWidth="40"></Button>
+            <Button Content="{ext:Localize XAML_None}" Command="{Binding ToggleAll}" CommandParameter="False" MinWidth="40"></Button>
+          </StackPanel>
+        
+        <ScrollViewer Margin="0 5 15 0" MaxHeight="300">
+          <ItemsControl Items="{Binding Options}">
+            <ItemsControl.ItemTemplate>
+              <DataTemplate>
+                <CheckBox Margin="0 5 0 0"
```

Comment:
Maybe instead of vanilla checkbox you could use the one scarab uses everywhere else
https://github.com/TheMulhima/Scarab/blob/master/Scarab/CustomControls/MenuCheckBox.axaml

Reply:
I don't know how to make it work with None and All buttons, color of the MenuCheckBox doesn't change when setting IsSelected

Reply:
I think you should be able to call notify property change on each instance of the is selected bool and it should update

Reply:
SetButtonColors is called by InternalOnPress, but not IsSelected